app/graphql/re_sip_e_be_schema.rb

```diff
@@ -42,4 +42,12 @@ def self.object_from_id(global_id, query_ctx)
     raise GraphQL::ExecutionError, "#{err.to_param}"
   end
 
+  rescue_from(Faraday::ServerError)  do |err, obj, args, ctx, field|
+    raise GraphQL::ExecutionError, "#{err.to_param}"
+  end
+
+  rescue_from(Faraday::ClientError)  do |err, obj, args, ctx, field|
+    raise GraphQL::ExecutionError, "#{err.to_param}"
+  end
+
```

Comment:
I love keeping all these rescue_from methods in the schema to blanket cover all our errors. I would think this helps future proof more iterations of the app as well.